lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -243,7 +246,7 @@ def verify
     res
   end
 
-  def add(source, dest_uri)
+  def add(source, dest_uri, _object_file = nil)
```

Comment:
Can this unused parameter be removed?

Reply:
I could swear I did that.  Obviously missed it.  Will do...